package.json

```diff
@@ -47,6 +47,7 @@
     "redux": "^4.2.0",
     "superjson": "^1.12.2",
     "update": "^0.7.4",
+    "usehooks-ts": "^2.9.1",
```

Comment:
I think there is no need for this package. We can use the visibility by tailwind to generate the following behaviour, unless there is a strong justification why you would like to use this package instead.

https://tailwindcss.com/docs/visibility